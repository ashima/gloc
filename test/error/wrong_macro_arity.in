#define F(A) (A)
void main() {
     F((3,2));
     F();
     F(5,3);
}
