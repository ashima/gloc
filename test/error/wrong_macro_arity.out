File 0, line 3, col 5 - 6:
too few macro arguments: expected 1, got 0

File 0, line 4, col 5 - 6:
too few macro arguments: expected 1, got 0

